Sming/Wiring/IpAddress.h

```diff
@@ -27,6 +27,12 @@
 #define LWIP_IP_ADDR_T const ip_addr_t
 #else
 typedef struct ip_addr ip_addr_t;
+typedef ip_addr_t ip4_addr_t;
+#define IP_ADDR4(IP, A, B, C, D) IP4_ADDR(IP, A, B, C, D)
```

Comment:
Why do we need IP_ADDR4? Looking at the IDF code it appears to be related to IPv6, in which case perhaps leave this change until it's needed? Or perhaps we should add v6 support at this stage, thus justifying the changes?

Reply:
> Why do we need IP_ADDR4?

Because it creates an address in which the IPv4 part is initialized using the octets that we provide.  In LWIP 1/2 with IP4 only it does not make much sense but in LWIP 2 with IPv4 + IPv6 it sets the correct bytes plus sets the type of the whole ip_addr_t structure to be IPv4 address. Or  you have meant something else?